Survey/Sources/Data/Repositories/SurveyRepository.swift

```diff
@@ -12,6 +12,8 @@ import Foundation
 protocol SurveyRepositoryProtocol: AnyObject {
 
     func getSurveyList(pageNumber: Int, pageSize: Int) -> Observable<[Survey]>
+
```

Comment:
too many line breaks 😄 
```suggestion
```

Reply:
^^ thank you Bliss, I just removed it at https://github.com/nkhanh44/ic-survey-ios/pull/50/commits/3c82aad7bd08e0a9ffcbf0cd2dad58974e1a4b15